src/db/connect.js

```diff
@@ -1,11 +1,13 @@
 const mongoose = require("mongoose");
 
-const connectDB = (url) => {
-  return mongoose.connect(url, {
+const connectDB = async(url) => {
+  const conn = await mongoose.createConnection(url, {
```

Comment:
Nitpick: might change the name here from conn to connection. Just so it's a little clearer as to what this function does.